ui/qt-app/Glue.hs

```diff
@@ -331,3 +322,9 @@ historyToUI ch = UiHistoryFace
   , historyPrevCommand = toPrevCommand ch
   }
 
+----------------------------------------------------------------------------
```

Comment:
This does not fit into any of our current faces. In face, we only have `langFace` exposed to the UI and I don't think `generateMnemonic` fits there.

Reply:
I think it should be a separate function (as it is), but defined inside the `Glue between the Wallet backend and Qt frontend` block (and exported right after `putWalletEventToUI`).